### ISSUE TITLE
Methods with ... argument forwarding don't show up in docs

### ISSUE THREAD

**Issue by svoop** [lsegal/yard#1439]
## Steps to reproduce

Given a source:

```ruby
class Foo

  # Forwards any kind of arguments
  def bar(...)
    baz(...)
  end

  # Prints a string
  #
  # @param string [String]
  def baz(string)
    puts string
  end
end
```

The documentation generated with `yard doc --one-file --no-private foo.rb` does not contain `bar`, however, if another signature than `(...)` is used, it will show up normally:

![Screenshot 2022-04-09 at 15 21 23](https://user-images.githubusercontent.com/7255/162576036-06debe0e-863f-4adc-afb1-696c79e33cb9.jpg)

## Environment details:

- OS: macOS 12.3.1
- Ruby version (`ruby -v`): 3.1.1
- YARD version (`yard -v`): 0.9.27

I have read the [Contributing Guide][contrib].

[contrib]: https://github.com/lsegal/yard/blob/main/CONTRIBUTING.md

**Comment by svoop** [comment]
In case this is not a bug but my bad: What docstring is required for methods with `(...)` signature to show up in the generated docs?

**Comment by ojab** [comment]
Looks like `...` method signature (ruby-2.7 feature) is not supported by yard parser, I'm getting
```
[error]: Unhandled exception in YARD::Handlers::Ruby::MethodHandler:
  in `lib/forward_test.rb`:1:

	1: def xxx(...)
```
here

**Comment by lsegal** [comment]
This should be fixed by #1431

**Comment by svoop** [comment]
@lsegal Great, thanks a lot!!
